sycl/include/sycl/string.hpp

```diff
@@ -0,0 +1,42 @@
+//==----------------- string.hpp - SYCL standard header file ---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include <iostream>
```

Comment:
This include is unnecessary.
In addition to that including `iostream` is forbidden -  
https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ContributeToDPCPP.md#using-iostream.

Reply:
Done. Thanks.